tests/test_graph.py

```diff
@@ -166,9 +166,8 @@ def test_serialize_graph_to_json(clear_default_graph, branching_graph):
 
 def test_serialize_graph_to_pickle(clear_default_graph, branching_graph):
     serialized = branching_graph.to_pickle()
-    deserialized = Graph.from_pickle(serialized).to_pickle()
-
-    assert serialized == deserialized
+    deserialized = Graph.from_pickle(serialized)
+    assert deserialized.to_json() == branching_graph.to_json()
```

Comment:
In python3.8, for some reason, the bytes object obtained when pickling, unpickling and pickling again changes in the first round trip, but stays static afterwards.
I used the serialized json rep as the closest thing to an equality check we have for graphs.